takari-lifecycle-plugin/src/main/java/io/takari/maven/plugins/compile/jdt/ProcessingEnvImpl.java

```diff
@@ -112,11 +112,11 @@ public Factory getFactory() {
     return _factory;
   }
 
-  public void hardReset() {
+  public void incrementalIterationReset() {
     reset();
     setErrorRaised(false);
 
-    ((FilerImpl) _filer).hardReset();
+    ((FilerImpl) _filer).incrementalIterationReset();
```

Comment:
"resets filer state between incremental compile iterations" does not seem to add much info to `incrementalIterationReset`, no?